public/index.html

```diff
@@ -3,8 +3,10 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="apple-itunes-app" content="app-id=1515759131">
     <link rel="shortcut icon" href="favicon.ico" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="manifest" href="manifest.json" />
```

Comment:
Just to keep everything unified, maybe you could try addin the `<meta>`/`<link>` tags from within the hook? It would keep everything in one place.